data/mods/Magiclysm/Spells/classless.json

```diff
@@ -17,6 +17,24 @@
     "max_duration": 1000000,
     "duration_increment": 2000
   },
+  {
+    "id": "invisibility",
+    "type": "SPELL",
+    "name": "Invisibility",
+    "description": "Allow light to pass through your body, rendering you invisible.",
+    "message": "Your body loses it's ability to reflect light, you are invisible!",
+    "valid_targets": [ "self" ],
+    "effect": "target_attack",
+    "effect_str": "invisibility",
+    "min_range": 1,
+    "max_range": 1,
+    "base_casting_time": 100,
+    "base_energy_cost": 400,
+    "energy_source": "MANA",
+    "min_duration": 100000,
```

Comment:
Starting duration of 16m40s(100000/100moves per second/60spm=16.667)? Much OP, very nerftime.

Reply:
Also missing max_level and difficulty

Reply:
Yes I will leave the number tweaking to KorGenT. :)

Reply:
in dnd this is a 4th level spell that lasts up to 10 rounds. (greater invisibility, since i assume the invisibility remains if you're fighting) LaVeyanFiend is right, hours is a very long time for something like this.

Reply:
from my testing it isn't hours though, I copied the values from night vision and it lasts like 10-15 minutes?

Reply:
Dark Sight/NV is level 0, that's why it's only 16m40s.
![image](https://user-images.githubusercontent.com/51099123/64367233-b6a15e80-cfe5-11e9-8def-061bb6f5092e.png)

Reply:
I believe they dont have the ability to level up?

Reply:
I think i tried the debug leveling at least. But I guess yea your looking at the min vs max durations. anyways. I'll put whatever values Korgent wants ;)